rpcs3/Emu/Cell/PPUInterpreter.cpp

```diff
@@ -1478,45 +1479,15 @@ bool ppu_interpreter_precise::VPKSHUS(ppu_thread& ppu, ppu_opcode_t op)
 {
 	const auto& a = ppu.vr[op.va];
 	const auto& b = ppu.vr[op.vb];
-	auto& d = ppu.vr[op.vd];
 
-	for (u8 i = 0; i < 8; i++)
+	// Detect saturation
+	if (const u64 _mask = 0xFF00FF00FF00FF00ULL;
```

Comment:
Please, don't abuse new `if` syntax like this. It lies outside of its purpose: const doesn't effectively have a lifetime, it's only strange to read.

Reply:
used constexpr without scope instead.